release-notes/Release-Notes-RC2.md

```diff
@@ -0,0 +1,123 @@
+# Release Notes
+
+## RC2 released 5/16/2016
+
+Full details on this release can be read about in this blog post:
+** [the blog title](http://someurl) **
+
+Platform support has been expanded for RC2
+
+* Red Hat Enterprise Linux 7.2
+* Debian 8.2 (8.2, 8.3, 8.4)+
+* Ubuntu 14.04 (16.04 support is coming in RTM)
+* OS X 10.11
+* Centos 7.1
+* Windows 7+ / Server 2012 R2+
+* Windows Nano Server TP5
+
+### Dependencies
+
+- [ ] Are there changes from RC1?
+- [ ] Should min versions of tools be included?
+
+.NET Core RC1 on Linux and OSX take dependencies on the following libraries:
+* libc, librt: I/O, process management, time, etc.
+* libunwind: call chain determination for exception handling and stack traces
+* gettext: resource strings
+* libicu: globalization
+* liblttng: runtime tracing
+* libuuid: guid generation
+* libcurl: HTTP
+* libssl & libcrypto: cryptography
```

Comment:
Sharing here as well.  I think the concept of a minimum version is a bit confusing.  I'd rather we just provide the names of packages we depend on for our supported distros.  To that end:

Debian/Ubuntu:

-	libunwind8
-	gettext
-	libicu52
-	liblttng-ust0
-	libuuid1
-	libcurl3
-	libssl1.0.0
-	zlib1g

CentOS/RHEL:

-	libunwind
-	gettext
-	libicu
-	libuuid
-	libcurl
-	openssl-libs
-	zlib

On CentOS and RHEL, LTTNG is not provided in the default package sources.  It is not necessary for .NET Core to run, but users will be unable to collect detailed performance traces.  If an end user desires it, she may install it from a third party source (http://packages.efficios.com/).